src/main/java/mate/academy/internetshop/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -71,4 +71,13 @@ public ShoppingCart update(ShoppingCart element) {
     public boolean delete(Long id) {
         return cartDao.delete(id);
     }
+
+    @Override
+    public double getTotalPrice(Long id) {
```

Comment:
whit this method you'll have to ask DB one more time even you would have this shoppingCart by that time

Reply:
changed. now this mrthod dont't call to DB